File: blog-dao/src/main/java/com/dao/CommentMapper.java
package com.dao;

import com.entity.Comment;

import java.util.List;

public interface CommentMapper {
    int deleteByPrimaryKey(Integer comid);

    int insert(Comment record);

    int insertSelective(Comment record);

    Comment selectByPrimaryKey(Integer comid);

    int updateByPrimaryKeySelective(Comment record);

    int updateByPrimaryKey(Comment record);


    //根据文章id查询出文章所对应的评论
    List<Comment> selectByAid(Integer aid);

}

File: blog-dao/src/main/java/com/dao/CommentMapper.kt
package com.dao

import com.entity.Comment

interface CommentMapper {
    fun deleteByPrimaryKey(comid: Int?): Int

    fun insert(record: Comment): Int

    fun insertSelective(record: Comment): Int

    fun selectByPrimaryKey(comid: Int?): Comment

    fun updateByPrimaryKeySelective(record: Comment): Int

    fun updateByPrimaryKey(record: Comment): Int
}